pkgs/applications/networking/cluster/helm/chart-testing/default.nix

```diff
@@ -36,8 +48,19 @@ buildGoModule rec {
       --bash <($out/bin/ct completion bash) \
       --zsh <($out/bin/ct completion zsh) \
       --fish <($out/bin/ct completion fish) \
+
+    wrapProgram $out/bin/ct --prefix PATH : ${lib.makeBinPath runtimeInputs}
   '';
 
+  runtimeInputs = [
+    coreutils
```

Comment:
Please inline.

Reply:
I borrowed this from https://github.com/NixOS/nixpkgs/blob/3357d39dc0c3ee6d4741a69c0921ae8957ea4ea1/pkgs/tools/networking/tinystatus/default.nix#L33, and this seems like the only sane way to allow customers to override `runtimeInputs` (say I need to provide a custom version of `kubectl`); is there a better way?

Reply:
IIRC that will not work in the current form. It could be done by overwriting the inputs or using finalAttrs which is overkill here.

Reply:
sgtm; so there is no way to make a trivially overridable thing, shame